exercises/acronym/canonical-data.json

```diff
@@ -1,44 +1,56 @@
 {
   "exercise": "acronym",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "cases": [
     {
       "description": "Abbreviate a phrase",
       "cases": [
         {
           "description": "basic",
           "property": "abbreviate",
-          "phrase": "Portable Network Graphics",
+          "input": {
+            "phrase": "Portable Network Graphics"
```

Comment:
This may want the same treatment that I have suggested in #1033.  Do we really need a "phrase" object since it is the only item with in "input"?

Reply:
don't care what y'all decide, but https://github.com/exercism/problem-specifications/issues/996#issuecomment-342740372 and https://github.com/exercism/problem-specifications/issues/996#issuecomment-342751835 are explicitly in favour of having `"input": {"phrase": "Portable Network Graphics"}` not `"input": "Portable Network Graphics"` therefore that is the only thing I support in https://github.com/exercism/problem-specifications/pull/998.